src/js/actions/transform.js

```diff
@@ -381,9 +385,9 @@ define(function (require, exports) {
                 positions: []
             },
             layerOneActions = _getMoveLayerActions
-                .call(this, document, layers.get(0), newPositions.get(0), payload.positions),
+                .call(this, document, layers.get(0), newPositions.get(0), payload.positions, false),
```

Comment:
Because I want that flag to be explicit across the codebase, just coding defensively.